commands/ci/debug.js

```diff
@@ -64,25 +64,48 @@ function* run (context, heroku) {
     cli.warn('to execute a build and configure the environment')
   }
 
-  let dyno = new Dyno({
-    heroku,
-    app: appSetup.app.id,
-    command: noSetup ? 'bash' : COMMAND,
-    'exit-code': true,
-    'no-tty': context.flags['no-tty'],
```

Comment:
This PR also removes this option which shouldn't have been added in the first place.